src/test/resources/features/QuestionBank/QuestionBank.feature

```diff
@@ -0,0 +1,114 @@
+@QuestionBank @Regression
+Feature: Question Bank
+
+  Background:
+    Given user prepare question bank request
+
+  @Negative @QuestionBank
+  Scenario: Create question bank without logging in
+    And user prepare auth request
```

Comment:
I think this can be moved to `Background`, unless such `prepare` statement is only used in a scenario, then it is fine to do such as this current implementation